apps/frontend/src/libs/components/protected-route/protected-route.tsx

```diff
@@ -0,0 +1,27 @@
+import { type FC, type ReactNode } from "react";
+import { Navigate } from "react-router-dom";
+
+import { AppRoute } from "~/libs/enums/app-route.enum.js";
+import { useAppSelector } from "~/libs/hooks/hooks.js";
+
+type ProtectedRouteProperties = {
```

Comment:
It is named as properties in other places
```suggestion
type Properties = {
```